ray-operator/DEVELOPMENT.md

```diff
@@ -66,8 +82,8 @@ kind load docker-image kuberay/operator:nightly
 
 # Step 6: Install KubeRay operator with the custom image via local Helm chart
 # (Path: helm-chart/kuberay-operator)
```

Comment:
If you want to add `../helm-chart/kuberay-operator` to the `helm install` command, we may need to remove this line.